subsys/bluetooth/host/hci_core.c

```diff
@@ -4771,6 +4794,58 @@ static void le_per_adv_sync_lost(struct net_buf *buf)
 		}
 	}
 }
+
+static void le_past_received(struct net_buf *buf)
+{
+	struct bt_hci_evt_le_past_received *evt =
+		(struct bt_hci_evt_le_past_received *)buf->data;
+	struct bt_le_per_adv_sync_synced_info sync_info;
+	struct bt_le_per_adv_sync_cb *listener;
+	struct bt_le_per_adv_sync *per_adv_sync;
+
+	if (evt->status) {
+		/* No sync created, don't notify app */
```

Comment:
Why not?

No cleanup that would have to be done?

Reply:
Not in the host, no. Receiving a PAST isn't an active thing in the host, but something that can merely happen at any time once configured. 

If the controller received a PAST, but somehow sends a status (not sure what the case would be), there's nothing we can do in the host as the receiver.